src/objects/button-group/button-group.scss

```diff
@@ -0,0 +1,33 @@
+@use "../../compiled/tokens/scss/size";
+
+/**
+ * Button Group layout object
+ * 1. Fallback for when Flexbox gap is not supported
+ */
+.o-button-group {
+  --gap: #{size.$button-group-gap-default};
+
+  display: flex;
+  flex-wrap: wrap;
+  margin: calc(var(--gap) / -2); /* 1 */
+
+  @supports (gap: 1em) {
```

Comment:
This will test if a `gap` property is supported, not if that property is supported for Flexbox. These are unfortunately slightly different in terms of support:

https://caniuse.com/flexbox-gap
https://caniuse.com/mdn-css_properties_gap_grid_context

Reply:
Ah, good point. I'll do some research to figure out if I can set up this `@supports` test differently. 🤔

Reply:
What benefit is there to have the `@supports` at all? It doesn't seem like progressive enhancement because it doesn't change the user experience AFAICT

Reply:
@calebeby Oh, you mean, just use margins and call it good? That's a good point, I could remove the use of `gap` and not worry about it.

Reply:
Yeah, and then down the road once gap for flexbox is more supported then we can just swap it out

Reply:
Good call. I'll do that and leave a comment. 👍

Reply:
Done. https://github.com/cloudfour/cloudfour.com-patterns/pull/1240/commits/23ed1884dee957eff209b902752f608f9938e825